src/main.rs

```diff
@@ -160,3 +152,25 @@ fn pass_to_sub(command: String) -> Result<(), MainError> {
         None => MainError::Pn(PnError::UnexpectedTermination { command }),
     })
 }
+
+fn create_path_env() -> OsString {
+    let bin_path = Path::new("node_modules").join(".bin");
+    if let Some(path) = env::var_os("PATH") {
+        bin_path
+            .pipe(std::iter::once)
+            .chain(env::split_paths(&path))
+            .pipe(env::join_paths)
+            .expect("Failed to prepend path") // TODO: propagate JoinPathError to main as a meaningful error message
```

Comment:
shouldn't this be finished?

Reply:
Are you asking "shouldn't this PR be merged?" or something else? Because you asked it as a review comment makes me confused.